src/emucore/Settings.hxx

```diff
@@ -162,6 +164,9 @@ class Settings
     */
     void setSize(const string& key, const int value1, const int value2);
 
+    /** Sets all of the ALE-specific default settings */
+    void setDefaultSettings();
```

Comment:
Do we want this method to be callable anywhere but from the constructor? If not, I would make it private.